<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>220</width>
				<height>223</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Formula</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_Shape" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout_5">
										<property name="spacing">
											<number>5</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="label_2">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Shape</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_3">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
												<property name="alignment">
													<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="Line_Shape">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_02" native="true">
						<layout class="QVBoxLayout" name="verticalLayout_2">
							<property name="spacing">
								<number>0</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QSplitter" name="splitter">
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
									<property name="handleWidth">
										<number>2</number>
									</property>
									<widget class="QPlainTextEdit" name="Text_PreFormula">
										<property name="sizePolicy">
											<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
												<horstretch>0</horstretch>
												<verstretch>0</verstretch>
											</sizepolicy>
										</property>
										<property name="minimumSize">
											<size>
												<width>0</width>
												<height>25</height>
											</size>
										</property>
										<property name="font">
											<font>
												<family>Consolas</family>
												<pointsize>9</pointsize>
											</font>
										</property>
										<property name="lineWrapMode">
											<enum>QPlainTextEdit::NoWrap</enum>
										</property>
									</widget>
									<widget class="QPlainTextEdit" name="Text_Formula">
										<property name="sizePolicy">
											<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
												<horstretch>0</horstretch>
												<verstretch>0</verstretch>
											</sizepolicy>
										</property>
										<property name="minimumSize">
											<size>
												<width>0</width>
												<height>25</height>
											</size>
										</property>
										<property name="font">
											<font>
												<family>Consolas</family>
												<pointsize>9</pointsize>
											</font>
										</property>
										<property name="lineWrapMode">
											<enum>QPlainTextEdit::NoWrap</enum>
										</property>
									</widget>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_03" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_3">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="Label_1">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>Variables can be assigned</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_5">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_6">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>x y z</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_04" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_6">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="Label_2">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>Variables can be readed</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_7">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="Label_Variables">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>20</height>
										</size>
									</property>
									<property name="text">
										<string>x y z</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_05">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_4">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QComboBox" name="Combo_CutType">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="currentIndex">
										<number>9</number>
									</property>
									<item>
										<property name="text">
											<string>Example 01</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 02</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 03</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 04</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 05</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 06</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 07</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 08</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 09</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 10</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 11</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 12</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 13</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 14</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 15</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>Example 16</string>
										</property>
									</item>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Fill">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Fill</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_07" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_071" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout" stretch="7,4">
										<property name="spacing">
											<number>2</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QCheckBox" name="Check_Secure">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Secure</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QCheckBox" name="Check_VariableN">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>$n</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Formulate">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Formulate</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>